scripts/collect-node-inventory.sh

```
@@ -0,0 +1,58 @@
+hostnamectl 2>/dev/null > "$out/hostnamectl.txt" || hostname > "$out/hostname.txt"
+cat /etc/os-release > "$out/os-release.txt"
+uname -a > "$out/uname.txt"
+lscpu > "$out/lscpu.txt"
+free -h > "$out/memory.txt"
+python3 --version > "$out/python-version.txt"
+lsblk -J -o NAME,MODEL,SIZE,TYPE,FSTYPE,MOUNTPOINTS > "$out/lsblk.json"
+
+if command -v lshw >/dev/null 2>&1; then
+  lshw_tmp="$out/lshw.json.tmp"
+  if [[ $EUID -eq 0 ]]; then
+    lshw -json > "$lshw_tmp" 2>/dev/null && mv "$lshw_tmp" "$out/lshw.json" || rm -f "$lshw_tmp"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)